circinus/docker-compose.yml

```diff
@@ -22,12 +22,44 @@ services:
       local:
         aliases:
           - circinus-apache
+  mongodb:
+    container_name: delphinus-mongodb
+    image: mongo:4.0
+    environment:
+       - MONGO_DATA_DIR=/data/db
+       - MONGO_LOG_DIR=/dev/null
+    volumes:
+       - mongodbdata:/data/db 
+    ports:
+      - "127.0.0.1:${ENV_MONGO_PORT:-27017}:27017"
+    command: mongod 
+    networks:
+      local:
+        aliases:
+          - delphinus-mongodb
```

Comment:
When I try running locally with `docker-compose up -d`, these delphinus-named containers are conflicting.  Please fix and test locally.

Reply:
Actually, they should be delphinus named since they are containers for the delphinus project inside the demo application.  The issue you are having is that you have delphinus running standalone alongisde the demo.  This is not a case that we should have in production.